bibs/Essen.json

```diff
@@ -27,6 +27,18 @@
             "location": 2,
             "status": 4
         },
+        "mediatypes": {
+            "0": "ALL",
+            "10": "MAP",
+            "11": "DVD",
+            "20": "AUDIOBOOK",
+            "24": "BLURAY",
+            "26": "GAME_CONSOLE",
+            "3": "CD_SOFTWARE",
+            "6": "CD_MUSIC",
+            "8": "SCORE_MUSIC",
+            "9": "BOARDGAME"
+        },
```

Comment:
I'll remove `ALL`. Here, the problem is, that all mediatypes have the same image `../icon/spacer.gif.S`, which is...no image.
With the current Config, Essen does not work if you filter the search by mediatypes, because all mediatypes have the same "spacer"-image.

I think you need to add an additional `if` [here](https://github.com/opacapp/opacclient/blob/499c78e0112b47d14c9e2da3ec8cf54ef75133f7/opacclient/libopac/src/main/java/de/geeksfactory/opacclient/apis/BiBer1992.java#L167), to the parsing for mediatypes, and use the inputs value.

I can't test this, because i don't have Android Studio installed.

Reply:
as I said above, the image itself is not used at all in the search form, just in the search results. So if search doesn't work in Essen, this is not the problem.